sycl/include/CL/sycl/handler.hpp

```diff
@@ -2663,6 +2672,16 @@ class __SYCL_EXPORT handler {
   /// The list of valid SYCL events that need to complete
   /// before barrier command can be executed
   std::vector<detail::EventImplPtr> MEventsWaitWithBarrier;
+  /// Host pipe name
+  std::string HostPipeName;
+  /// Host pipe host pointer
```

Comment:
Great to have comments!
But they could be more precise than just reading the variable name. I have no idea about what is the host pointer.